src/main/resources/config/config-template.conf

```diff
@@ -0,0 +1,26 @@
+input.osm: {
```

Comment:
I assume optionality due to different sources that will be available in the future?

Reply:
Nah. You mandatorily need osm data, right? Everything within it should be optional (with boundary conditions, that only one of them is set).